packages/internet-header/src/assets/i18n/translations.ts

```diff
@@ -39,4 +39,9 @@ export const translations = {
     fr: 'Naviguer sur post.ch',
     it: 'Navigate su post.ch',
   },
+  'Intermediary links': {
+    de: 'Zwischenverbindungen',
+    fr: 'Liens intermédiaires',
+    it: 'Collegamenti intermedi',
```

Comment:
```suggestion
  'Jump links': {
    de: 'Sprunglinks',
    fr: 'Liens de saut',
    it: 'Collegamenti di salto',
```

Reply:
We could also go with "skip links" in english. It should be pretty well known by users.

Reply:
Sorry for all the changes,I didn't understand at the beginning what these translations were all about.